src/clj/game/cards-hardware.clj

```diff
@@ -341,7 +342,20 @@
                                 (swap! state update-in [side :prompt] rest)
                                 (when-let [run (:run @state)]
                                   (when (and (:ended run) (empty? (get-in @state [:runner :prompt])) )
-                                    (handle-end-run state :runner)))))}]}
+                                    (handle-end-run state :runner)))))}
+                {:once :per-turn
+                 :label "Move a previously accessed card to bottom of R&D"
+                 :effect (effect (resolve-ability
+                                   {; only allow targeting cards that were accessed this turn -- not perfect, but good enough?
+                                    :choices {:req #(some (fn [c] (= (:cid %) (:cid c)))
+                                                          (map first (turn-events state side :access)))}
+                                    :msg (msg "move " (:title target) " to the bottom of R&D")
+                                    :effect (req (move state :corp target :deck)
+                                                 (tag-runner state :runner 1)
+                                                 (swap! state update-in [side :prompt] rest)
+                                                 (when-let [run (:run @state)]
+                                                   (when (and (:ended run) (empty? (get-in @state [:runner :prompt])))
+                                                     (handle-end-run state :runner))))} card nil))}]}
```

Comment:
Note that this won't adjust agenda points of a Runner who ditches a News Team or Shi.Kyu, nor of a Corp who has a QPM taken away. Adjusting points could be kind of onerous, so maybe at a minimum a toast if the targeted card has a non-zero number of agenda points? I can easily imagine people forgetting to adjust agenda points manually in these cases mid-game.